pygmt/helpers/decorators.py

```diff
@@ -445,7 +445,7 @@ def fmt_docstring(module_func):
     - J = projection
     - R = region
     <BLANKLINE>
-    """
+    """  # noqa: D410,D411
```

Comment:
Since [D410](https://docs.astral.sh/ruff/rules/no-blank-line-after-section) (a blank line after section headings) is enabled in 5d983e633bffe46b324fc55f2c4ebf7880f7d3d2, this `noqa: D410, D411` can be removed (and you'll need to run formatting after).

Reply:
Actually, this line is required. If removing `noqa: D410, D411`, the codes will be formatted with the following diff:
```diff
diff --git a/pygmt/helpers/decorators.py b/pygmt/helpers/decorators.py
index 57bd0f900..b44dd7fec 100644
--- a/pygmt/helpers/decorators.py
+++ b/pygmt/helpers/decorators.py
@@ -425,6 +425,7 @@ def fmt_docstring(module_func):
     <BLANKLINE>
     My nice module.
     <BLANKLINE>
+
     Parameters
     ----------
     data : str, numpy.ndarray, pandas.DataFrame, xarray.Dataset, or geo...
@@ -445,7 +446,7 @@ def fmt_docstring(module_func):
     - J = projection
     - R = region
     <BLANKLINE>
-    """  # noqa: D410,D411
+    """
     filler_text = {}

     if hasattr(module_func, "aliases"):
```
Then the doctest fails, because only one blank line is expected, but two blank lines are given (one is `<BLANKLINE>`, another one is the newly added blank line).

Reply:
We could just remove the `<BLANKLINE>` from L427 no?

Reply:
Yeah, looks like we'll need to keep it, since doctest takes an empty line to mean the end of the expected output according to https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized